modules/tile-converter/src/3d-tiles-converter/3d-tiles-converter.ts

```diff
@@ -48,6 +49,7 @@ export default class Tiles3DConverter {
     this.sourceTileset = null;
     this.attributeStorageInfo = null;
     this.workerSource = {};
+    this.message = 'Tile converter does not work in browser, only in node js environment';
```

Comment:
I am not sure constructor of  Tiles3DConverter is a good place for such messages. 
I think separate constant will be sufficient.